bespin/option_spec/bespin_specs.py

```diff
@@ -302,8 +284,8 @@ def stack_spec(self):
             , stack_json = valid_stack_json(default="{config_root}/{_key_name_1}.json")
             , stack_yaml = valid_stack_yaml(default="{config_root}/{_key_name_1}.yaml")
 
-            , params_json = valid_params_json(default="{config_root}/{environment}/{_key_name_1}-params.json")
-            , params_yaml = valid_params_yaml(default="{config_root}/{environment}/{_key_name_1}-params.yaml")
```

Comment:
because of this line here https://github.com/atward/bespin/blob/8e3c5b856737f60a8066eadc40ff0e407402c560/bespin/option_spec/bespin_specs.py#L56 which triggers if https://github.com/atward/bespin/blob/8e3c5b856737f60a8066eadc40ff0e407402c560/bespin/option_spec/bespin_specs.py#L43 is False